schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE invoices (
+    "id"  SERIAL PRIMARY KEY,
+    "total_amount" numeric,
+    "generated_at" date,
+    "payed_at" date,
+    "medical_history_id" int
+);
+
+ALTER TABLE invoices ADD FOREIGN KEY ("medical_history_id") REFERENCES medical_history("id");
+
+
+CREATE TABLE invoice_items (
+    "id"  SERIAL PRIMARY KEY,
+    "unit_price" numeric,
+    "quantity" int,
+    "total_price" numeric,
+    "invoice_id" int,
+    "treatment_id" int
+);
+
+ALTER TABLE invoice_items ADD FOREIGN KEY ("treatment_id") REFERENCES treatments("id");
+ALTER TABLE invoice_items ADD FOREIGN KEY ("invoice_id") REFERENCES invoices("id");
+
+
+CREATE TABLE medical_history (
+    "id"  SERIAL PRIMARY KEY,
+    "admitted_at" date,
+    "patient_id" int,
+    "status" varchar
+);
+
+ALTER TABLE medical_history ADD FOREIGN KEY ("patient_id") REFERENCES patients("id");
+
+
+CREATE TABLE patients (
+    "id"  SERIAL PRIMARY KEY,
+    "name" varchar,
+    "date_of_birth" date
+);
+
+CREATE TABLE treatments (
+    "id"  SERIAL PRIMARY KEY,
+    "type" varchar,
+    "name" varchar
+);
+
+
+
```

Comment:
- [ ] Amazing job so far but I think it would be great if you add indexes to the foreign keys you created above since the indexes are very important to improve the performance of the database queries so kindly add indexes in all the FK 💪🏻🤩⭐💯 you can check this [lesson](https://www.w3schools.com/sql/sql_ref_create_index.asp) to learn more about how to make indexes 💯


![image](https://github.com/yemidada/database-performance-audit/assets/58553711/53463886-3249-404a-b074-f4170a7f320f)

Reply:
Thank you for a constructive feedback @Reem-lab. Working on it.